src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Reload/ProjectReloadInterceptor.cs

```diff
@@ -4,6 +4,7 @@
 using System.Collections.Immutable;
 using System.ComponentModel.Composition;
 using Microsoft.Build.Construction;
+using Microsoft.VisualStudio.ProjectSystem.VS;
```

Comment:
The IProjectReloadInterceptor is in your VS namespace? Did it need anything VS specific when you moved it?